### DISCUSSION TITLE
ValueError on creating stock item

### DISCUSSION THREAD

**Discussion by HannesMorre** [inventree/InvenTree#5745] [Q&A]
I created a part with a specified supplier part pack quantity and supplier price break. But when I try to add a stock item of these parts, I get this error:

![image](https://github.com/inventree/InvenTree/assets/71696745/216a639e-5a33-4dcb-9eba-4d96640b6aa9)

These are the part and supplier part:
![image](https://github.com/inventree/InvenTree/assets/71696745/3a8f32c7-6a96-4aec-881d-c12f38a0a49e)
![image](https://github.com/inventree/InvenTree/assets/71696745/e30d4d86-6fca-495a-96d1-c77b5d1840f0)

Am I doing something wrong or is this a bug?

**Comment by HannesMorre** [comment]
I tried to reproduce this on the demo page, but there it works just fine.

**Comment by SchrodingersGat** [comment]
What version are you running? This may have been already fixed

**Reply by HannesMorre** [reply]
0.12.7

**Comment by SchrodingersGat** [comment]
Ok, looks like this was fixed on master but not stable - see https://github.com/inventree/InvenTree/pull/5751
